test/ui/ui_unit.js

```diff
@@ -783,13 +778,32 @@ describe('UI', () => {
             .toBe(true);
       });
 
+      it('container receives all the available statistics', () => {
```

Comment:
Oh, I just noticed this, but your test names are technically formatted wrong.
When the results of the test are written, Jasmine will include the name of the `it` after the name of the `describe`. So this would be output as:
`statistics context menu container receives all the available statistics`

If this was named something like "displays all available statistics", the result would look much nicer:
`statistics context menu displays all available statistics`

Incidentally, I think that's why the function is called `it`... it's supposed to read like a sentence ("it displays all available statistics").